CHANGELOG.md

```diff
@@ -1604,7 +1604,7 @@ New modules
   Data.List.NonEmpty.Relation.Unary.All
   ```
 
-* A small library for heterogenous equational reasoning on vectors:
+* A small library for heterogeneous equational reasoning on vectors:
   ```
   Data.Vec.Properties.Heterogeneous
   ```
```

Comment:
Just to double check, is this module not pushed to `master` when merging #1668?

Reply:
Ha! Good catch! I think the module got removed in the final edit, but not the `CHANGELOG` entry...
... and I never said that I was not myself responsible for some of these spelling errors. ;-)